swap_meet/vendor.py

```diff
@@ -1,2 +1,57 @@
 class Vendor:
-    pass
+    def __init__ (self, inventory = None):
+        if not inventory:
+            inventory = []
+        self.inventory = inventory
+    
+    def add(self, add_item):
+        self.inventory.append(add_item)
+        return add_item
+
+    def remove(self, remove_item):
+        if remove_item in self.inventory:
+            self.inventory.remove(remove_item)
+            return remove_item
+        else: 
+            return False
```

Comment:
Consider refactoring to first check for the edge case to enhance readability:

```suggestion
        if remove_item not in self.inventory:
            return False
            
        self.inventory.remove(remove_item)
        return remove_item
```